scripts/update_transaction_amountInTxnCurrency.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * This scripts populates Transaction.netAmountInTxnCurrency based on historic fxrate
```

Comment:
I'm hesitant to populate this data based on historic fxdata that doesn't match host's own books - their accounts won't match exactly and will create a headache for them to track differences per line item.

Do you know how many transactions fall in the category where host and group currencies differ? And for how many of them we aren't the host?